dev/com.ibm.ws.ejbcontainer.core/src/com/ibm/ejs/container/TimerNpRunnable.java

```diff
@@ -191,29 +191,47 @@ private void doWork() {
         }
     }
 
+    /**
```

Comment:
javadoc is wrong ... "Getter for bean id" is not correct; it is the timer implementation.

Also, if you have an @return there should be text after it to say what is returned.  For simple getters, the method description and @return description are often redundant, so you could have just one or the other.  An empty @return will produce a javadoc warning, so if present, there should be text after it.

Reply:
Got it, I updated the comment and removed @return here and in other places.